src/app/app.imports.ts

```diff
@@ -34,5 +35,7 @@ export const APP_IMPORTS = [
   StoreModule.provideStore(rootReducer),
   STORE_DEV_TOOLS_IMPORTS,
   StoreDevToolsModule,
-  TransferHttpModule
+  TransferHttpModule,
+  WebsiteModule
```

Comment:
You still need to work on routing concepts. In lazy loaded routes WebsiteModule should not be imported to app module. So why is this still imported?